File: 1598. Crawler Log Folder.java
class Solution {
    public int minOperations(String[] logs) {
        int deepth = 0;
        for(String log : logs) {
            if(log.startsWith("../")) {
                if(deepth > 0) {
                    deepth--;
                }
            } else if(log.startsWith("./")) {
                
            } else {
                deepth++;
            }
        }
        return deepth;
    }
}

File: 1598. Crawler Log Folder.kt
class Solution {
    fun minOperations(logs: Array<String>): Int {
        var depth = 0
        for(log in logs) {
            if(log == "../") {
                if(depth > 0) {
                    depth--
                }
            } else if(log == "./") {

            } else {
                depth++
            }
        }
        return depth
    }
}
